### DISCUSSION TITLE
IRAF V2.16.1+ on RHEL8 ecl.e not compiled

### DISCUSSION THREAD

**Discussion by AstroBacon** [iraf-community/iraf#186] [Q&A] [answered]
I'm trying to do a systemwide installation of V2.16.1+ on RHEL8. It seems to go ok, but when I try to run the tests, I get:
IRAF shell ecl.e not found
stat: cannot stat '/iraf/iraf/bin.linux64/ecl.e': No such file or directory
When I check, the ecl.e is indeed not there. I'm using the linux64 irafarch when I do the make.  Does ecl not work on the linux64?

Also, it makes a dead link to itself in /iraf/iraf/bin.linux64/bin.linux64 -> bin.linux64. That's not a problem, but I thought I'd mention it.

**Comment by AstroBacon** [comment, marked as answer]
I was missing the readline package. I installed that and recompiled IRAF. Now ecl.e compiles.

**Reply by Vital-Fernandez** [reply]
I am having the same issue. Could you share the command to instal the readline package?

Thank you.

**Reply by AstroBacon** [reply]
On RHEL8
yum install readline readline-devel

**Reply by Vital-Fernandez** [reply]
Thank you very much for your reply. I guess that was not my issue as the error message is still there :).

**Reply by olebole** [reply]
The installation document (`INSTALL.md`) described which packages need to be installed on Fedora or RedHat. Wasn't this sufficient?
@Vital-Fernandez Unfortunately the installation does not stop on the first error. You should grep the `build.log` for error messages (search for `error` or so).
